### PR TITLE
Add new languages

### PR DESCRIPTION
**Pull request by RainbowDashLabs**
Add some new languages.

I dont speak any of this, so hopefully someone will complain xD

### CODE REVIEW

File: src/main/resources/locale_pt_PT.properties [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+command.roles.sub.managerRole.current=A actual função de gestor é %ROLE%.
+command.roles.sub.managerRole.noRole=Sem função de gestor definida.
+command.roles.sub.managerRole.set=A função de gestor definida como %ROLE%.
+command.roles.sub.remove=Remover uma função de reputação.
+command.roles.sub.remove.notARepRole=Esta função não é uma função de reputação.
+command.roles.sub.remove.removed=Remover um papel %ROLE%.
+command.scan.canceled=Scan cancelado.
+command.scan.canceling=Tentativa de cancelar o scan.
+command.scan.completed=Varredura concluída.
+command.scan.description=Scan a channel for reputations.
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:110]: Looks untranslated

DeepL suggestion:
```suggestion
command.scan.description=Digitalizar um canal em busca de reputações.
```

File: src/main/resources/locale_pt_PT.properties [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+command.scan.error.history=Desaparecimento da permissão para histórico de mensagens
+command.scan.error.queueFull=Todos os scans estão actualmente em execução. Por favor aguarde até que uma ranhura esteja livre.
+command.scan.error.running=Já está a decorrer uma tarefa nesta guild. Por favor aguarde até que a tarefa esteja terminada.
+command.scan.progress=Progress: %PERCENT%
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:114]: Looks untranslated

DeepL suggestion:
```suggestion
command.scan.progress=Progresso: %PERCENT%
```

File: src/main/resources/locale_pt_PT.properties [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+command.scan.result=Scanned %SCANNED% messages.\nAdded %HITS% reputation.
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:115]: Looks untranslated

DeepL suggestion:
```suggestion
command.scan.result=Mensagens digitalizadas %SCANNED%.\nAdicionado %HITS% reputação.
```

File: src/main/resources/locale_pt_PT.properties [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+command.scan.scheduling=Agramação do seu scan. O tempo necessário estimado até à conclusão é: %DURATION%
+command.scan.sub.cancel=Cancelar um scan em execução.
+command.scan.sub.cancel.noTask= Nenhuma tarefa está em execução para esta guilda.
+command.setup.description=Executar uma configuração básica do bot.
+command.setup.dialog.channels=Por favor introduza todos os canais onde os utilizadores têm permissão para receber reputação.
+command.setup.dialog.continueToProceed=Please press "$word.continue$" to proceed.
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:121]: Looks untranslated

DeepL suggestion:
```suggestion
command.setup.dialog.continueToProceed=Por favor prima "$word.continue$" para prosseguir.
```

File: src/main/resources/locale_pt_PT.properties [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+command.setup.dialog.loadDefaults=Carregar palavras de agradecimento por defeito para as suas línguas.
+command.setup.dialog.managerRole=Please write one role which should be allowed to change the settings of the bot.
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:123]: Looks untranslated

DeepL suggestion:
```suggestion
command.setup.dialog.managerRole=Por favor, escreva uma função que deve ser permitida para alterar as definições do bot.
```

File: src/main/resources/locale_pt_PT.properties [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+command.setup.dialog.roles=Por favor defina alguns papéis de reputação.\nEstes papéis serão dados a um utilizador quando este atingir a quantidade especificada de reputação.\nDefina-os como <$words.role$> <$words.reputation$>.\nQuando tiver definido todos os seus papéis prima o botão.
+command.setup.dialog.rolesFormat=Por favor defina-os como <$words.role$> <$words.reputation$>
+command.setup.dialog.selectLanguage=Please select your language.
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:126]: Looks untranslated

DeepL suggestion:
```suggestion
command.setup.dialog.selectLanguage=Por favor, seleccione a sua língua.
```
Alternative DeepL suggestion:
```suggestion
command.setup.dialog.selectLanguage=Seleccione a sua língua.
```

File: src/main/resources/locale_pt_PT.properties [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+command.setup.dialog.setupComplete=Setup complete!
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:127]: Looks untranslated

DeepL suggestion:
```suggestion
command.setup.dialog.setupComplete=Configuração completa!
```

File: src/main/resources/locale_pt_PT.properties [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+command.setup.dialog.starting=Arrancar setup.
+command.setup.dialog.welcome=Bem-vindos à configuração do bot de reputação.
+command.thankwords.description=Configurar palavras.de.agradecimento
+command.thankwords.error.patternNotFound=Pattern não existe.
+command.thankwords.sub.add=Add another pattern. Padrões são interpretados como Regex. Um padrão não pode conter espaços.
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:132]: Looks partially untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.add=Acrescentar outro padrão. Padrões são interpretados como Regex. Um padrão não pode conter espaços.
```
Alternative DeepL suggestion:
```suggestion
command.thankwords.sub.add=Adicionar outro padrão. Padrões são interpretados como Regex. Um padrão não pode conter espaços.
```

File: src/main/resources/locale_pt_PT.properties [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+command.thankwords.sub.add.added=Adicionar padrão %REGEX%.
+command.thankwords.sub.check=Cheque se uma mensagem daria reputação.
+command.thankwords.sub.check.confidence=Confidence Score: %SCORE%
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:135]: Looks untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.check.confidence=Nota de Confiança: %SCORE%
```

File: src/main/resources/locale_pt_PT.properties [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+command.thankwords.sub.check.match.answer=Aresposta Match
+command.thankwords.sub.check.match.fuzzy=Fuzzy Match
+command.thankwords.sub.check.match.mention=Mention Match
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:138]: Looks untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.check.match.mention=Menção Match
```

File: src/main/resources/locale_pt_PT.properties [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+command.thankwords.sub.check.match.noMatch=No Match
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:139]: Looks untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.check.match.noMatch=Sem correspondência
```

File: src/main/resources/locale_pt_PT.properties [hunk 12]
```diff
@@ -0,0 +1,196 @@
...
+command.thankwords.sub.check.notConfident=Confidence Score is to low. Não será dada uma reputação.
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:140]: Looks partially untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.check.notConfident=A pontuação de confiança é muito baixa. Não será dada uma reputação.
```

File: src/main/resources/locale_pt_PT.properties [hunk 13]
```diff
@@ -0,0 +1,196 @@
...
+command.thankwords.sub.check.reference=Reference Message: [Saltar](%URL%)
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:141]: Looks partially untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.check.reference=Mensagem de referência: [Saltar](%URL%)
```

File: src/main/resources/locale_pt_PT.properties [hunk 14]
```diff
@@ -0,0 +1,196 @@
...
+command.thankwords.sub.check.result=Donatador: %DONATOR%%NReceptor: %RECEIVER%
+command.thankwords.sub.list=Listar todos os padrões
+command.thankwords.sub.list.list=Following Pattern are active:
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:144]: Looks untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.list.list=Os seguintes Padrões estão activos:
```

File: src/main/resources/locale_pt_PT.properties [hunk 15]
```diff
@@ -0,0 +1,196 @@
...
+command.thankwords.sub.loadDefault=Load default thankswords for a language.
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:145]: Looks untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.loadDefault=Carregar palavras de agradecimento por defeito para uma língua.
```

File: src/main/resources/locale_pt_PT.properties [hunk 16]
```diff
@@ -0,0 +1,196 @@
...
+command.thankwords.sub.loadDefault.added=Added the following thankwords:
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:146]: Looks untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.loadDefault.added=Acrescentadas as seguintes palavras de agradecimento:
```

File: src/main/resources/locale_pt_PT.properties [hunk 17]
```diff
@@ -0,0 +1,196 @@
...
+command.thankwords.sub.loadDefault.available=Línguas disponíveis:
+command.thankwords.sub.remove=Delete a pattern
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:148]: Looks untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.remove=Eliminar um padrão
```

File: src/main/resources/locale_pt_PT.properties [hunk 18]
```diff
@@ -0,0 +1,196 @@
...
+command.thankwords.sub.remove.removed=Removed pattern %PATTERN%.
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:149]: Looks untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.remove.removed=Padrão removido %PATTERN%.
```

File: src/main/resources/locale_pt_PT.properties [hunk 19]
```diff
@@ -0,0 +1,196 @@
...
+conversation.cancel=cancelar
+conversation.canceled=Conversation canceled.
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:151]: Looks untranslated

DeepL suggestion:
```suggestion
conversation.canceled=A conversa foi cancelada.
```

File: src/main/resources/locale_pt_PT.properties [hunk 20]
```diff
@@ -0,0 +1,196 @@
...
+conversation.exit=saída
+conversation.finished=Conversation finished.
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:153]: Looks untranslated

DeepL suggestion:
```suggestion
conversation.finished=Conversa terminada.
```
Alternative DeepL suggestion:
```suggestion
conversation.finished=Conversa concluída.
```

File: src/main/resources/locale_pt_PT.properties [hunk 21]
```diff
@@ -0,0 +1,196 @@
...
+conversation.inProgress=Já está em curso uma conversação neste canal. Pode sair desta conversa digitando $conversation.exit$ ou $conversation.cancel$.
+conversation.start=Iniciando uma nova conversação.\n Você pode deixar a conversa a qualquer momento digitando $conversation.exit$ ou $conversation.cancel$.
+error.commandNotFound=Command não encontrado.
+error.invalidArguments= Argumentos inválidos
+error.invalidChannel=Invalid Channel
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:158]: Looks untranslated

DeepL suggestion:
```suggestion
error.invalidChannel=Canal Inválido
```

File: src/main/resources/locale_pt_PT.properties [hunk 22]
```diff
@@ -0,0 +1,196 @@
...
+error.invalidMessage=Mensagem inválida
+error.invalidNumber=Número inválido
+error.invalidRegex=Invalid regex pattern.
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:161]: Looks untranslated

DeepL suggestion:
```suggestion
error.invalidRegex=Padrão regex inválido.
```
Alternative DeepL suggestion:
```suggestion
error.invalidRegex=Padrão de regex inválido.
```

File: src/main/resources/locale_pt_PT.properties [hunk 23]
```diff
@@ -0,0 +1,196 @@
...
+error.invalidRole=Invalid Role
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:162]: Looks untranslated

DeepL suggestion:
```suggestion
error.invalidRole=Papel inválido
```

File: src/main/resources/locale_pt_PT.properties [hunk 24]
```diff
@@ -0,0 +1,196 @@
...
+error.missingPermission=Sinto falta da permissão %PERM% para funcionar correctamente.
+error.missingPermissionChannel= Isto deve-se à definição de permissões de anulação no canal %CHANNEL%.
+error.missingPermissionGuild=É devido a definições de permissão globais.
+error.noValidChannel=Não são fornecidos canais válidos.
+error.notABoolean=%INPUT% is not a boolean (true|false).
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:167]: Looks untranslated

DeepL suggestion:
```suggestion
error.notABoolean=%INPUT% não é um booleano (true|false).
```

File: src/main/resources/locale_pt_PT.properties [hunk 25]
```diff
@@ -0,0 +1,196 @@
...
+listener.reaction.confirmation=%DONOR% deu reputação a %RECEIVER%.
+listener.reaction.removal=Reputação dada por %DONOR% foi removida.
+message.welcome=Perdão por me ter convidado.\n Você pode iniciar uma configuração fácil com o comando /setup.
+messages.destruction= Esta mensagem autodestruir-se-á em %MIN% minuto.
+thankType.answer=Resposta
+thankType.fuzzy=Fuzzy
+thankType.mention=Menção
+thankType.reaction=Reacção
+word.continue=Continuar
+word.done=Done
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:185]: Looks untranslated

DeepL suggestion:
```suggestion
word.done=Feito
```

File: src/main/resources/locale_pt_PT.properties [hunk 26]
```diff
@@ -0,0 +1,196 @@
...
+word.skip=Skip
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:186]: Looks untranslated

DeepL suggestion:
```suggestion
word.skip=Saltar
```

File: src/main/resources/locale_pt_PT.properties [hunk 27]
```diff
@@ -0,0 +1,196 @@
...
+words.code=Código
+words.jumpMarker=[%TARGET%](%URL%)
+words.language=Langauge
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:189]: Looks untranslated

DeepL suggestion:
```suggestion
words.language=Língua
```

File: src/main/resources/locale_pt_PT.properties [hunk 28]
```diff
@@ -0,0 +1,196 @@
...
+words.level=Nível
+words.message=Mensagem
+words.name=Nome
+words.refMessage=Mensagem de referência
+words.reputation=Reputação
+words.role=Role
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:195]: Looks untranslated

DeepL suggestion:
```suggestion
words.role=Papel
```

File: src/main/resources/locale_es_ES.properties [hunk 29]
```diff
@@ -0,0 +1,196 @@
...
+thankType.answer=Respuesta
+thankType.fuzzy=Fuzzy
+thankType.mention=Mención
+thankType.reaction=Reacción
+word.continue=Continuar
+word.done=Hecho
+word.skip=Saltar
+words.code=Código
+words.jumpMarker=[%TARGET%](%URL%)
+words.language=Langauge
```
Comment (Taucher2003) [inline src/main/resources/locale_es_ES.properties:189]: Looks untranslated

DeepL suggestion:
```suggestion
words.language=Lengua
```

File: src/main/resources/locale_fr_FR.properties [hunk 30]
```diff
@@ -0,0 +1,196 @@
...
+error.notYourEmbed=Ce n'est pas ton embed !
+error.prefixTooLong=Le préfixe ne peut comporter que 3 caractères.
+error.roleAccess=On ne peut pas interagir avec le rôle %ROLE%. Veuillez ajuster la hiérarchie.
+error.userNotFound=Impossible de trouver cet utilisateur.
+listener.messages.request.descr=Hey, tu viens de remercier quelqu'un, veux-tu remercier quelqu'un en particulier ? Si oui, clique sur le bouton ci-dessous.
+listener.messages.request.descrThank=Merci d'avoir donné de la réputation à quelqu'un. Tu peux donner %MORE% plus de réputation.
+listener.messages.request.title=Remerciements confirmation
+listener.reaction.confirmation=%DONOR% a donné de la réputation à %RECEIVER%.
+listener.reaction.removal=Réputation donnée par %DONOR% a été supprimée.
+message.welcome=Merci de m'avoir invité.\NTu peux démarrer une installation facile avec la commande /setup.
```
Comment (Taucher2003) [inline src/main/resources/locale_fr_FR.properties:178]:
```suggestion
message.welcome=Merci de m'avoir invité.\nTu peux démarrer une installation facile avec la commande /setup.
```

File: src/main/resources/locale_fr_FR.properties [hunk 31]
```diff
@@ -0,0 +1,196 @@
...
+messages.destruction=Ce message s'autodétruira dans %MIN% minute.
+thankType.answer=Réponse
+thankType.fuzzy=Fuzzy
+thankType.mention=Mention
+thankType.reaction=Réaction
+word.continue=Continue
```
Comment (Taucher2003) [inline src/main/resources/locale_fr_FR.properties:184]: Looks untranslated

DeepL suggestion:
```suggestion
word.continue=Continuer
```

File: src/main/resources/locale_fr_FR.properties [hunk 32]
```diff
@@ -0,0 +1,196 @@
...
+word.skip=Skip
+words.code=Code
+words.jumpMarker=[%TARGET%](%URL%)
+words.language=Langue
+words.level=Niveau
+words.message=Message
+words.name=Nom
+words.refMessage=Message de référence
+words.reputation=Réputation
+words.role=Role
```
Comment (Taucher2003) [inline src/main/resources/locale_fr_FR.properties:195]: Looks untranslated

DeepL suggestion:
```suggestion
words.role=Rôle
```

**Review by Taucher2003 — CHANGES_REQUESTED** [review]

**Review by yannicklamprecht — COMMENTED** [review]
Not deep(l) enough in the language to add constructive feedback here.

File: src/main/resources/locale_fr_FR.properties [hunk 33]
```diff
@@ -0,0 +1,196 @@
...
+command.prefix.sub.reset=Réinitialise le préfixe
+command.prefix.sub.set=Définir le préfixe
+command.prune.description=Élimine les utilisateurs non valides ou un utilisateur spécifique.
+command.prune.sub.guild=Remove reptutation of users which are no longer on the guild.
+command.prune.sub.guild.done=Enlever %AMOUNT% des utilisateurs.
+command.prune.sub.guild.started=Démarrer la suppression des utilisateurs qui ne sont plus dans la guilde.
+command.prune.sub.user=Remove reputation of an user from the server.
+command.prune.sub.user.removed=L'utilisateur sera supprimé.
+command.repSettings.description=Recevoir les paramètres de réputation actuels.
+command.repSettings.embed.descr=Age maximal des messages : %MAX_AGE%\nMin Messages : %MIN_MESSAGES%\nRéaction de réputation : %REACTION%\nRéaction par réaction : %REACTION_ACTIVE%\nRéaction par réponse : %ANSWER_ACTIVE%\nRéaction par mention : %MENTION_ACTIVE%\nRéaction par recherche floue : %FUZZY_ACTIVE%\nRéputation en temps de refroidissement : %COOLDOWN%%RÉPUTATION PAR RECHERCHE FLOUE : %FUZZY_ACTIVE
```
Comment (Taucher2003) [inline src/main/resources/locale_fr_FR.properties:60]:
```suggestion
command.repSettings.embed.descr=Age maximal des messages : %MAX_AGE%\nMin Messages : %MIN_MESSAGES%\nRéaction de réputation : %REACTION%\nRéaction par réaction : %REACTION_ACTIVE%\nRéaction par réponse : %ANSWER_ACTIVE%\nRéaction par mention : %MENTION_ACTIVE%\nRéaction par recherche floue : %FUZZY_ACTIVE%\nRéputation en temps de refroidissement : %COOLDOWN%\nRéputation par mention floue : %FUZZY_ACTIVE
```

File: src/main/resources/locale_pt_PT.properties [hunk 34]
```diff
@@ -0,0 +1,196 @@
...
+command.thankwords.sub.add.added=Adicionar padrão %REGEX%.
+command.thankwords.sub.check=Cheque se uma mensagem daria reputação.
+command.thankwords.sub.check.confidence=Nota de Confiança: %SCORE%
+command.thankwords.sub.check.match.answer=Aresposta Match
+command.thankwords.sub.check.match.fuzzy=Fuzzy Match
+command.thankwords.sub.check.match.mention=Menção Match
+command.thankwords.sub.check.match.noMatch=Sem correspondência
+command.thankwords.sub.check.notConfident=A pontuação de confiança é muito baixa. Não será dada uma reputação.
+command.thankwords.sub.check.reference=Mensagem de referência: [Saltar](%URL%)
+command.thankwords.sub.check.result=Donatador: %DONATOR%%NReceptor: %RECEIVER%
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:142]: Double %?

File: src/main/resources/locale_es_ES.properties [hunk 35]
```diff
@@ -0,0 +1,196 @@
...
+command.roles.sub.remove.removed=Eliminar rol %ROLE%.
+command.scan.canceled=Escaneo cancelado.
+command.scan.canceling=Intentando cancelar el escaneo.
+command.scan.completed=Escaneo completado.
+command.scan.description=Escanear un canal en busca de reputaciones.
+command.scan.error.history=Falta de permiso para el historial de mensajes.
+command.scan.error.queueFull=Se están ejecutando demasiados escaneos actualmente. Por favor, espere hasta que haya un espacio libre.
+command.scan.error.running=Se está ejecutando una tarea en este gremio. Por favor, espere a que termine.
+command.scan.progress=Progreso: %PERCENT%
+command.scan.result=Mensajes escaneados %SCANNED%.\NSe ha añadido reputación %HITS%.
```
Comment (Taucher2003) [inline src/main/resources/locale_es_ES.properties:115]:
```suggestion
command.scan.result=Mensajes escaneados %SCANNED%.\nSe ha añadido reputación %HITS%.
```

File: src/main/resources/locale_es_ES.properties [hunk 36]
```diff
@@ -0,0 +1,196 @@
...
+command.scan.scheduling=Programando el escaneo. El tiempo estimado necesario hasta la finalización es: %DURATION%
+command.scan.sub.cancel=Cancelar una exploración en curso.
+command.scan.sub.cancel.noTask=No hay ninguna tarea en ejecución para este gremio.
+command.setup.description=Realizar una configuración básica del bot.
+command.setup.dialog.channels=Introducir todos los canales en los que los usuarios pueden recibir reputación.
+command.setup.dialog.continueToProceed=Por favor, pulsa "$palabra.continue$" para continuar.
+command.setup.dialog.loadDefaults=Cargar las palabras de agradecimiento por defecto para sus idiomas.
+command.setup.dialog.managerRole=Por favor, escriba un rol que pueda cambiar la configuración del bot.
+command.setup.dialog.roles=Por favor, defina algunos roles de reputación.\NEstos roles se darán a un usuario cuando alcance la cantidad de reputación especificada.\NDefínalos como <$words.role$> <$words.reputation$>.\NCuando haya definido todos sus roles pulse el botón.
```
Comment (Taucher2003) [inline src/main/resources/locale_es_ES.properties:124]:
```suggestion
command.setup.dialog.roles=Por favor, defina algunos roles de reputación.\nEstos roles se darán a un usuario cuando alcance la cantidad de reputación especificada.\nDefínalos como <$words.role$> <$words.reputation$>.\nCuando haya definido todos sus roles pulse el botón.
```

File: src/main/resources/locale_fr_FR.properties [hunk 37]
```diff
@@ -0,0 +1,196 @@
...
+command.scan.result=Messages scannés %SCANNED%.\nAjouté %HITS% de réputation.
+command.scan.scheduling=Planifie ton analyse. Le temps nécessaire estimé jusqu'à l'achèvement est de : %DURATION%.
+command.scan.sub.cancel=Annule un scan en cours.
+command.scan.sub.cancel.noTask=Aucune tâche n'est en cours pour cette guilde.
+command.setup.description=Exécute une configuration de base du robot.
+command.setup.dialog.channels=Veuillez saisir tous les canaux où les utilisateurs sont autorisés à recevoir de la réputation.
+command.setup.dialog.continueToProceed=Veuillez appuyer sur "$word.continue$" pour continuer.
+command.setup.dialog.loadDefaults=Charge les mots de remerciement par défaut pour tes langues.
+command.setup.dialog.managerRole=Veuillez écrire un rôle qui doit être autorisé à modifier les paramètres du robot.
+command.setup.dialog.roles=Veuillez définir des rôles de réputation.\NCes rôles seront attribués à un utilisateur lorsqu'il atteindra le niveau de réputation spécifié.\NDéfinis-les comme suit : <$words.role$> <$words.reputation$>.\NLorsque tu as défini tous tes rôles, appuie sur le bouton.
```
Comment (Taucher2003) [inline src/main/resources/locale_fr_FR.properties:124]:
```suggestion
command.setup.dialog.roles=Veuillez définir des rôles de réputation.\nCes rôles seront attribués à un utilisateur lorsqu'il atteindra le niveau de réputation spécifié.\nDéfinis-les comme suit : <$words.role$> <$words.reputation$>.\nLorsque tu as défini tous tes rôles, appuie sur le bouton.
```

File: src/main/resources/locale_fr_FR.properties [hunk 38]
```diff
@@ -0,0 +1,196 @@
...
+command.thankwords.sub.loadDefault.added=Ajoute les mots de remerciement suivants :
+command.thankwords.sub.loadDefault.available=Les langues disponibles :
+command.thankwords.sub.remove=Supprimer un motif
+command.thankwords.sub.remove.removed=Removed pattern %PATTERN%.
+conversation.cancel=annuler
+conversation.canceled=Conversation annulée.
+conversation.exit=sortir
+conversation.finished=Conversation terminée.
+conversation.inProgress=Une conversation est déjà en cours dans ce canal. Tu peux quitter cette conversation en tapant $conversation.exit$ ou $conversation.cancel$.
+conversation.start=Démarre une nouvelle conversation.\NTu peux quitter la conversation à tout moment en tapant $conversation.exit$ ou $conversation.cancel$.
```
Comment (Taucher2003) [inline src/main/resources/locale_fr_FR.properties:155]:
```suggestion
conversation.start=Démarre une nouvelle conversation.\nTu peux quitter la conversation à tout moment en tapant $conversation.exit$ ou $conversation.cancel$.
```

**Review by yannicklamprecht — APPROVED** [review]

File: src/main/resources/locale_es_ES.properties [hunk 39]
```diff
@@ -0,0 +1,196 @@
...
+command.scan.error.queueFull=Se están ejecutando demasiados escaneos actualmente. Por favor, espere hasta que haya un espacio libre.
+command.scan.error.running=Se está ejecutando una tarea en este gremio. Por favor, espere a que termine.
+command.scan.progress=Progreso: %PERCENT%
+command.scan.result=Mensajes escaneados %SCANNED%.\nSe ha añadido reputación %HITS%.
+command.scan.scheduling=Programando el escaneo. El tiempo estimado necesario hasta la finalización es: %DURATION%
+command.scan.sub.cancel=Cancelar una exploración en curso.
+command.scan.sub.cancel.noTask=No hay ninguna tarea en ejecución para este gremio.
+command.setup.description=Realizar una configuración básica del bot.
+command.setup.dialog.channels=Introducir todos los canales en los que los usuarios pueden recibir reputación.
+command.setup.dialog.continueToProceed=Por favor, pulsa "$palabra.continue$" para continuar.
```
Comment (Taucher2003) [inline src/main/resources/locale_es_ES.properties:121]: A key has been translated
```suggestion
command.setup.dialog.continueToProceed=Por favor, pulsa "$word.continue$" para continuar.
```

File: src/main/resources/locale_fr_FR.properties [hunk 40]
```diff
@@ -0,0 +1,196 @@
...
+command.scan.scheduling=Planifie ton analyse. Le temps nécessaire estimé jusqu'à l'achèvement est de : %DURATION%.
+command.scan.sub.cancel=Annule un scan en cours.
+command.scan.sub.cancel.noTask=Aucune tâche n'est en cours pour cette guilde.
+command.setup.description=Exécute une configuration de base du robot.
+command.setup.dialog.channels=Veuillez saisir tous les canaux où les utilisateurs sont autorisés à recevoir de la réputation.
+command.setup.dialog.continueToProceed=Veuillez appuyer sur "$word.continue$" pour continuer.
+command.setup.dialog.loadDefaults=Charge les mots de remerciement par défaut pour tes langues.
+command.setup.dialog.managerRole=Veuillez écrire un rôle qui doit être autorisé à modifier les paramètres du robot.
+command.setup.dialog.roles=Veuillez définir des rôles de réputation.\nCes rôles seront attribués à un utilisateur lorsqu'il atteindra le niveau de réputation spécifié.\nDéfinis-les comme suit : <$words.role$> <$words.reputation$>.\nLorsque tu as défini tous tes rôles, appuie sur le bouton.
+command.setup.dialog.rolesFormat=Veuillez les définir comme <$mots.rôle$> <$mots.réputation$>.
```
Comment (Taucher2003) [inline src/main/resources/locale_fr_FR.properties:125]: Keys have been translated
```suggestion
command.setup.dialog.rolesFormat=Veuillez les définir comme <$words.role$> <$words.reputation$>.
```

**Review by dlsf — APPROVED** [review]
There are sometimes too many spaces but otherwise it looks fine

**Review by Taucher2003 — APPROVED** [review]
Looks good now

**Review by Taucher2003 — COMMENTED** [review]
Looks good except for some missing translations

File: src/main/resources/locale_fr_FR.properties [hunk 41]
```diff
@@ -0,0 +1,207 @@
...
+command.log.sub.donated=Recevoir la réputation donnée d'un utilisateur.
+command.log.sub.message=Recevoir le journal d'un message.
+command.log.sub.received=Obtient la réputation reçue d'un utilisateur.
+command.prefix.changed=Préfixe défini sur %PREFIX%.
+command.prefix.description=Montrer le préfixe
+command.prefix.show=Le préfixe actuel est %PREFIX%.
+command.prefix.sub.reset=Réinitialise le préfixe
+command.prefix.sub.set=Définir le préfixe
+command.prune.description=Élimine les utilisateurs non valides ou un utilisateur spécifique.
+command.prune.sub.guild=Remove reptutation of users which are no longer on the guild.
```
Comment (Taucher2003) [inline src/main/resources/locale_fr_FR.properties:59]: Looks untranslated

DeepL suggestion:
```suggestion
command.prune.sub.guild=Supprimer la reptation des utilisateurs qui ne sont plus sur la guilde.
```

File: src/main/resources/locale_fr_FR.properties [hunk 42]
```diff
@@ -0,0 +1,207 @@
...
+command.prune.sub.guild.done=Enlever %AMOUNT% des utilisateurs.
+command.prune.sub.guild.started=Démarrer la suppression des utilisateurs qui ne sont plus dans la guilde.
+command.prune.sub.user=Remove reputation of an user from the server.
```
Comment (Taucher2003) [inline src/main/resources/locale_fr_FR.properties:62]: Looks untranslated

DeepL suggestion:
```suggestion
command.prune.sub.user=Supprimer la réputation d'un utilisateur du serveur.
```

File: src/main/resources/locale_fr_FR.properties [hunk 43]
```diff
@@ -0,0 +1,207 @@
...
+command.thankwords.description=Configurer Thankwords
+command.thankwords.error.patternNotFound=Le modèle n'existe pas.
+command.thankwords.sub.add=Ajouter un autre motif. Les motifs sont interprétés comme des Regex. Un motif ne peut pas contenir d'espaces.
+command.thankwords.sub.add.added=Ajouté le motif %REGEX%.
+command.thankwords.sub.check=Vérifie si un message donne une réputation.
+command.thankwords.sub.check.confidence=Score de confiance : %SCORE%.
+command.thankwords.sub.check.match.answer=Mise en correspondance des réponses
+command.thankwords.sub.check.match.fuzzy=Correspondance floue
+command.thankwords.sub.check.match.mention=Mention Match
+command.thankwords.sub.check.match.noMatch=No Match
```
Comment (Taucher2003) [inline src/main/resources/locale_fr_FR.properties:149]: Looks untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.check.match.noMatch=Pas de correspondance
```
Alternative DeepL suggestion:
```suggestion
command.thankwords.sub.check.match.noMatch=Pas de match
```

File: src/main/resources/locale_fr_FR.properties [hunk 44]
```diff
@@ -0,0 +1,207 @@
...
+command.thankwords.sub.check.notConfident=Le score de confiance est trop faible. Une réputation ne sera pas donnée.
+command.thankwords.sub.check.reference=Message de référence : [Sauter](%URL%)
+command.thankwords.sub.check.result=Donateur : %DONATOR%\nReceveur : %RECEIVER%.
+command.thankwords.sub.list=Liste de tous les motifs
+command.thankwords.sub.list.list=Les motifs suivants sont actifs :
+command.thankwords.sub.loadDefault=Charge les mots de remerciement par défaut pour une langue.
+command.thankwords.sub.loadDefault.added=Ajoute les mots de remerciement suivants :
+command.thankwords.sub.loadDefault.available=Les langues disponibles :
+command.thankwords.sub.remove=Supprimer un motif
+command.thankwords.sub.remove.removed=Removed pattern %PATTERN%.
```
Comment (Taucher2003) [inline src/main/resources/locale_fr_FR.properties:159]: Looks untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.remove.removed=Modèle supprimé %PATTERN%.
```

**Review by Taucher2003 — CHANGES_REQUESTED** [review]

**Review by Taucher2003 — APPROVED** [review]

**Pull request merged**